dotCMS/src/main/java/com/dotmarketing/quartz/job/DeleteOldClickstreams.java

```diff
@@ -28,7 +28,7 @@ public void run(JobExecutionContext jobContext)
 		now.add(Calendar.DATE, -(days-1));
 		
 		try {
-			if(DbConnectionFactory.isOracle()){				
+			if(DbConnectionFactory.isOracle()){	// todo: this should be encapsulated on an util class

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)